### DISCUSSION TITLE
Problems with the Way We Use Data and GIS

### DISCUSSION THREAD

**Discussion by jsxu315** [albertkun/21S-ASIAAM-191A#44] [Past Thinking Caps]
When collecting data, the categories that we use can be affected by our own biases; for example, in the Lab1 shapefile table, there are two fields for race, ‘other’ and ‘mixed’. The choice to use these two categories instead of, for example, letting people write in/check all groups they belong to, can erase some minorities’ backgrounds. For example, if a report shows that people in the ‘other’ group experienced a disproportionate percentage of hate crimes, that data can’t be used to enact meaningful change without intensive follow-up — it’s hard to combat ‘other’-based discrimination without knowing where it comes from and the social dynamics behind it. 

Even non-other/mixed categories can encompass a broad range of backgrounds and experiences so that data collected on them doesn’t accurately reflect what a population experiences or needs. For example, the Census defines “White” as “a person having origins in any of the original peoples of Europe, the Middle East, or North Africa” and “Asian” as including “Cambodia, China, India, Japan, Korea, Malaysia, Pakistan, the Philippine Islands, Thailand, and Vietnam.” (1) There’s a vast range of experiences among the people who check these boxes, which means that some will inevitably be drowned out when schools and policymakers make decisions based on the data. 

How data is collected and by whom can also affect the overall dataset; for example, if a survey or the Census asked people about their citizenship status, non-citizens might be wary of answering census collectors entirely and therefore be undercounted. 

Another (unrelated to the above) issue that piqued my interest was the transformation of subjective data, such as pain assessments, into objective assessments. While this article (2) isn’t directly applicable to statistical methods, it brought up some interesting ideas to me about dealing with self-reported, subjective data. For example, the way a person expresses and reports their pain can be influenced by a person’s background and experiences, e.g. two people who actually have the same stomach condition might report different pain levels because one has been through much worse pain before, even though they need care equally urgently. In the context of gathering and interpreting data, surveys that ask people to report their opinions based on a scale like “none, mild, moderate, severe” might require other data to paint a more accurate picture that can be used to enact change. 

(1) https://www.census.gov/topics/population/race/about.html
(2) https://www.sciencedirect.com/science/article/pii/S0277953619303120

**Comment by vcta99** [comment]
Great points about collecting data in its relation to race via census tracking. In general, I think data analysis must always strike a balance between being standardized (in order to draw meaningful insights) and being nuanced (as to not stereotype or lump dissimilar groups together). Racializing Middle Eastern and North African folks as white especially seems to be a pretty controversial with a long history behind it. From what I can gather, it seems that these communities have been actively advocating for their own racial category (MENA) on the 2020 census, and that their categorization as "White" diminished visibility of Arab American communities in order to promote misconceptions of a white majority in the U.S. I've pulled a quote from an article Arab media news outlet below: 

_“Why? If you take Arabs out of the white category, it’s going to drop,” Khalaf told Al Jazeera. “Almost to the point that it’s going to be a white minority and God forbid that should happen when you basically have government officials who are white extremists.”_

https://www.aljazeera.com/news/2020/4/1/why-is-there-no-mena-category-on-the-2020-us-census

**Reply by gu-ana** [reply]
I never thought of it like that but certain people would definitely be outraged by the fact that white people are no longer the majority. In fact, when Latinos officially outnumbered Non-Latino Whites in California back in 2014, I remember people still yelling "don't speak Spanish you live in America" and yet they live in a state where the majority of the population are minorities.
